blocks/ads-block/features/ads/ad-helper.js

```diff
@@ -8,6 +7,7 @@ export const getSizemapBreakpoints = (props) => {
 	return !breakpoints
 		? []
 		: [
+				// setup these multiple breakpoints in the theme styles breakpoints
```

Comment:
See: https://github.com/WPMedia/arc-themes-blocks/pull/1458

Reply:
I interpreted this need wrong.  This is not for styling?  Ignore this PR...

Reply:
this seem to be for populating `arcads` `sizemap` values.  Need to dig into `arcads` to find out what these are for.

Reply:
I think I have a solve for this, can talk about it in dev huddle

Reply:
ref: https://github.com/washingtonpost/arcads#size-mapping

Reply:
thanks for your pr

Reply:
is this ok to resolve @nschubach ?